packages/starknet-snap/src/ui/fragments/JsonDataUI.tsx

```diff
@@ -0,0 +1,21 @@
+import type { SnapComponent } from '@metamask/snaps-sdk/jsx';
+
+import { toJson } from '../../utils';
+import { RowUI } from './RowUI';
+
+export type JsonDataUIProps = {
+  label: string;
+  data: any;
+};
+/**
+ * Build a row component with the JSON data.
+ *
+ * @param params - The parameters.
+ * @param params.data - The JSON data.
+ * @param params.label - The label.
+ * @returns A row component with the JSON data.
+ */
+export const JsonDataUI: SnapComponent<JsonDataUIProps> = ({
```

Comment:
the comment still not fixed, and why some components in the root components/, some in fragments/?

Reply:
this is fixed in next PR

Reply:
i think we should fix this in this PR, because it doesnt make sense to do it in other PR, when u know u are going to change it, and they have no different in terms of feat

Reply:
noted